extras/Makefile

```diff
@@ -1,9 +1,9 @@
 all: sdist bdist_wheel
 
 sdist:
-	python setup.py sdist
+	python -m build --sdist
 
 bdist_wheel:
-	python setup.py bdist_wheel
+	python -m build --wheel
 
 .PHONY : sdist bdist_wheel
```

Comment:
According to `python -m build --help`, passing `--wheel` means that it won't create the wheel from the sdist, but from the source folder. The advantage of creating the wheel from the sdist is that it can verify that the sdist can be used to build wheels:

> By default, a source distribution (sdist) is built from `{srcdir}` and a binary distribution (wheel) is built from the sdist. This is recommended as it will ensure the sdist can be used to build wheels.
> 
> Pass `-s`/`--sdist` and/or `-w`/`--wheel` to build a specific distribution. If you do this, the default behavior will be disabled, and all artifacts will be built from `{srcdir}` (even if you combine `-w`/`--wheel` with `-s`/`--sdist`, the wheel will be built from `{srcdir}`).

In light of this, I suggest that we ditch this `extras/Makefile` file and replace `make -f extras/Makefile` in `docs/internals/howto-release-django.txt` with `python -m build` which is one simple command - the makefile is superfluous. It also makes those instructions flow better, as they contain `python -m pip install build twine` already, so using `make` is just obscuring what we're doing.